build_scripts/windows/README.md

```diff
@@ -0,0 +1,27 @@
+Building the Windows MSI Installer
+========================
+
+This document provides instructions on creating the MSI installer.
+
+Prerequisites
+-------------
+
+1. Turn on the '.NET Framework 3.5' Windows Feature (required for WIX Toolset).
+2. Install 'Microsoft Build Tools 2015'.
+    https://www.microsoft.com/download/details.aspx?id=48159
+3. You need to have curl.exe, unzip.exe and msbuild.exe available under PATH.
+4. Install 'WIX Toolset build tools' following the instructions below.
```

Comment:
make these steps as a ps1 file, so we will be easier to create a workflow from it

Reply:
leave it to the next pr